packages/manager/telecom-styles/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@ovh-ux/manager-telecom-styles",
```

Comment:
It would be great to add some meta (`description` and `keywords` for instance).

Reply:
@antleblanc : Like this ?

```json 
  "description" : "Legacy styles from ovh-manager-telecom",
  "keywords": [
    "control",
    "ovh",
    "panel",
    "style",
    "telecom"
  ],
```

Reply:
LGTM